errors/InputValidationError.js

```diff
@@ -0,0 +1,14 @@
+// Import base error
+const { BaseIsomerError } = require('./BaseError')
+
+class InputNameConflictError extends BaseIsomerError {
```

Comment:
For the isomerCMS project, we've been trying to switch to a different custom error system where the error is solely identified by the HTTP error code, to avoid a situation where we have way too many custom errors as we did in Homer. In keeping with this new strategy, this should be a `ConflictError` with an argument in the constructor for the error message.

To standardize the error messages, we can define the types of `ConflictError` messages and export them. For example, we can have:

```
const inputNameConflictErrorMsg = (fileName) => `A file with ${fileName} already exists.`

class ConflictError extends BaseIsomerError { ... }

module.exports = {
  inputNameConflictErrorMsg,
  ConflictError,
}
```

and then throw the error like this:

```
throw new ConflictError(inputNameConflictErrorMsg(fileName))
```

Reply:
Resolved with https://github.com/isomerpages/isomercms-backend/pull/78/commits/10ab10383ce3b458a66e66b0a49b2688647f6229